core/lib/zksync_core/src/api_server/web3/mod.rs

```diff
@@ -7,12 +7,14 @@ use serde::Deserialize;
 use tokio::sync::{oneshot, watch, RwLock};
 use tower_http::{cors::CorsLayer, metrics::InFlightRequestsLayer};
 
+use chrono::NaiveDateTime;
 use std::{net::SocketAddr, sync::Arc, time::Duration};
 use tokio::task::JoinHandle;
 
 use zksync_dal::{ConnectionPool, StorageProcessor};
 use zksync_health_check::{HealthStatus, HealthUpdater, ReactiveHealthCheck};
 use zksync_types::{api, MiniblockNumber};
+use zksync_web3_decl::types::Filter;
 use zksync_web3_decl::{
```

Comment:
Bikeshedding: Although we don't enforce any particular `use` grouping for now, IMO, it makes sense to at least have a single `use` group for external crates in order to improve readability.

Reply:
That was more because I haven't formatted the code properly